src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2402,6 +2402,11 @@ properties:
             type: string
             fits_keyword: S_PHOTOM
             blend_table: True
+          pixel_replace:
```

Comment:
not a big deal, but could go ahead and just spell out the full name to `pixel_replacement`